lib/src/node-sass/render.ts

```diff
@@ -183,6 +187,8 @@ function newRenderResult(
     }
   }
 
+  const includedFiles = loadedUrls.map(url => url.replace('file://', ''));
```

Comment:
This needs to handle file URLs properly on Windows, too.

```suggestion
  const includedFiles = loadedUrls.map(url => {
    return url.startsWith('file://') ? fileURLToPath(url) : url;
  });
```